### PR TITLE
Vue-Partners page

### PR DESCRIPTION
**Pull request by FRFlor**
## Purpose 

Implement Partners page.

image

### CODE REVIEW

File: src/partners/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        logo: vehikl.png
+        proficiencies:
+            - VueJs
+            - Laravel
+            - Node
+            - Team Augmentation
+        location: Ontario, Canada
+        languages:
+            - English
+        url: www.vehikl.com?utm_source=vue_partners_page
```
Comment (yyx990803) [inline src/partners/index.md]: Just realized we probably want to have two separate URLs, one for display and one for actual linking (it looks cleaner without the campaign tracking params)

File: themes/vue/layout/partners-page.ejs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  <p>
+    Vue Partners are premium shops that provide first-class Vue consulting and development.
+  </p>
+
+  <h2>Active Partners</h2>
+
+  <% for (let partner of page.partners_list) { %>
+    <div class="partner">
+      <div class="logo">
+        <img src="<%- url_for(`/images/${partner.logo}`) %>" alt="<%= partner.name %>">
```
Comment (grgur) [inline themes/vue/layout/partners-page.ejs]: Would it be possible to use a remote logo? E.g. something on Cloudinary?

File: src/partners/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                url: 'https://github.com/vehikl'
+            -
+                name: Twitter
+                class: twitter
+                url: 'https://twitter.com/vehikl'
+            -
+                name: Instagram
+                class: instagram
+                url: 'https://www.instagram.com/vehiklteam/'
+---
```
Comment (grgur) [inline src/partners/index.md]: What do you think about adding a short description for each partner?

File: themes/vue/layout/partners-page.ejs (hunk 2 above)
Comment (grgur) [reply themes/vue/layout/partners-page.ejs]: Note: using an external image may help track statistics.

Comment (yyx990803) [reply themes/vue/layout/partners-page.ejs]: Yeah we can make that work.

**Comment by yyx990803** [comment]
This is merged into master but I temporarily hid the navigation link until we are ready. In the meanwhile, new partners can be added via separate PRs.

Suggestions by @grgur are landed in fc7eeac6.

**Pull request merged**